Don't install the systemd exceptions file; we have our own (on-reboot)
cleanup of temp mounts which wouldn't be affected.

--- CMakeLists.txt.orig	2020-05-17 18:36:57 UTC
+++ CMakeLists.txt
@@ -57,7 +57,9 @@ target_include_directories(kio-fuse PRIVATE ${FUSE3_IN
 target_compile_definitions(kio-fuse PRIVATE FUSE_USE_VERSION=31 ${FUSE3_CFLAGS_OTHER})
 target_link_libraries(kio-fuse PRIVATE Qt5::Core KF5::KIOCore ${FUSE3_LIBRARIES} ${FUSE3_LDFLAGS})
 install(TARGETS kio-fuse DESTINATION ${KDE_INSTALL_FULL_LIBEXECDIR})
-install(FILES kio-fuse-tmpfiles.conf DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/tmpfiles.d)
+if(LINUX)
+	install(FILES kio-fuse-tmpfiles.conf DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/tmpfiles.d)
+endif()
 kdbusaddons_generate_dbus_service_file("kio-fuse -f" org.kde.KIOFuse ${KDE_INSTALL_FULL_LIBEXECDIR})
 
 feature_summary(WHAT ALL FATAL_ON_MISSING_REQUIRED_PACKAGES)
